openerp/addons/base/ir/ir_cron.py

```diff
@@ -87,7 +87,7 @@ def method_direct_trigger(self, cr, uid, ids, context=None):
             context={}
         cron_obj = self.browse(cr, uid, ids, context=context)
         for cron in cron_obj:
-            self._callback(cr, uid, cron_obj.model, cron_obj.function, cron_obj.args, cron_obj.id)
+            self._callback(cr, SUPERUSER_ID, cron_obj.model, cron_obj.function, cron_obj.args, cron_obj.id)
```

Comment:
Hello, 

Indeed that was a mistake (That you reported even before I tested the branch ^^)

Thank you.